pkg/process/checks/container.go

```diff
@@ -146,7 +158,7 @@ func (c *ContainerCheck) Cleanup() {}
 
 // chunkContainers formats and chunks the ctrList into a slice of chunks using a specific number of chunks.
 func chunkContainers(containers []*model.Container, chunks int) [][]*model.Container {
-	perChunk := (len(containers) / chunks) + 1
+	perChunk := int(math.Round(float64(len(containers)) / float64(chunks)))
```

Comment:
It would technically be possible for this to be 0 now. Is there any risk there?

Reply:
Good point! When perchunk is 0 and the len(containers) is not 0, it will try to append everything to one same chunk as the length of the chunk can never be 0 (i.e there won't be chunking as in the original code). I shall fix this by using math.Ceil.